gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/__.java

```diff
@@ -810,6 +810,20 @@ protected __() {
     }
 
     /**
+     * @see GraphTraversal#skip(long)
+     */
+    public static <A> GraphTraversal<A, A> skip(final long n) {
```

Comment:
We call the variable `final long limit` for `limit()`. Perhaps call `n` `skip` so it has the same look and feel. `skip(final long skip)`. Likewise for your `Scope` version.